src/Helpers/Capsule.php

```diff
@@ -365,4 +367,22 @@ public function getType(): string
     {
         return '';
     }
+
+    public function registerBlocksAndRepeatersViewPaths(): void
+    {
+        $resourcePath = $this->getConfig()['views_path'] ?? 'resources/views/admin';
```

Comment:
Hey @antonioribeiro 

Was having a look at this. But I feel we do not need the admin part here.

As capsules are already specific to twill, we could just have `resources/views/[blocks|repeaters]`

What do you think?

Reply:
I believe we do, @haringsrob , we are following this structure on some of our Capsules: https://github.com/area17/twill-google-recaptcha/tree/1.x/src/resources/views.

But how can we make this flexible in order to support any structure? Some other people might structure their Capsules other ways...